src/sentry/api/serializers/models/event.py

```diff
@@ -346,13 +349,32 @@ class SqlFormatEventSerializer(EventSerializer):
     Applies formatting to SQL queries in the serialized event.
     """
 
+    formatted_sql_cache: Dict[str, str] = {}
```

Comment:
You shouldn't declare this at the class level. This will cause the dict to persist between instances of `SqlFormatEventSerializer`. You'll need to define it in `__init__` to avoid this

Reply:
Ah right of course! Thanks for catching this, still getting used to python